tests/optional_scalars2_generated.rs

```diff
@@ -18,69 +18,74 @@ pub mod optional_scalars {
   extern crate flatbuffers;
   use self::flatbuffers::EndianScalar;
 
-#[allow(non_camel_case_types)]
-#[repr(i8)]
-#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
-pub enum OptionalByte {
-  None = 0,
-  One = 1,
-  Two = 2,
-
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Please, add a `#[repr(transparent)]` to that and similar structs. It would allow to construct a `[Color]` slice from `[u8]` slice or cast `&u8` into `&Color` without UB.

Reply:
Done